src/lib/sw.js

```diff
@@ -192,13 +192,18 @@ workboxRouting.registerRoute(normalMatch, async ({url}) => {
   }
   const partial = await response.json();
 
-  // Our target browsers all don't mind if we just place <title> in the middle of the document.
-  // This is far simpler than trying to find the right place in <head>.
   const meta = partial.offline ? `<meta name="offline" value="true" />` : '';
-  const output = layoutTemplate.replace(
-    '%_CONTENT_REPLACE_%',
-    meta + `<title>${partial.title || ''}</title>` + partial.raw,
-  );
+  const title = `<title>${partial.title || 'web.dev'}</title>`;
+  const rssHref = partial.rss || '/feed.xml';
+  const rssTitle =
+    partial.rss !== '/feed.xml'
+      ? `${partial.title} on web.dev`
+      : 'web.dev feed';
+  const rss = `<link rel="alternate" href="${rssHref}" type="application/atom+xml" data-title="${rssTitle}" />`;
+
+  const output = layoutTemplate
+    .replace(/<title\>.*\<\/title\>/, `${meta}\n${title}\n${rss}`)
```

Comment:
@samthor does this look good for replacing the content in the head? It seems like we need to use a proper tag in the head rather than the `%_HEAD_REPLACE_%` in order to add the tags to the `sw-partial-layout.partial`.

Reply:
Something something don't parse HTML with regexp.

This is kind of going down the rabbit hole here. What was the problem with just inserting `<title>` and friends inline in the document? Every browser we ship to doesn't seem to care if we include these at the "wrong" place.

Reply:
@robdodson should we just put it in the wrong place?